.flake8

```diff
@@ -0,0 +1,33 @@
+[flake8]
```

Comment:
Put this in `setup.cfg`, to the existing `[flake8]` section

Reply:
done with 9209785b76efe21c56eb3e7df135d09c0028e3ad